index.bs

```diff
@@ -12521,6 +12518,19 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+The internal \[[PreventExtensions]] method of every [=platform object=] |O|
```

Comment:
OK, so the phrasing of the other methods is confusing.

Take for example [[[DefineOwnProperty](https://heycam.github.io/webidl/#defineownproperty)]].

It first states:

> When the internal [[DefineOwnProperty]] method of a platform object O that implements an interface which supports indexed or named properties is called with property key P and Property Descriptor Desc, the following steps must be taken:

So I read the rest of the algorithm as only applying to the case where O implements an interface which supports indexed or named properties.

It's actually not explicit from the algorithm itself whether the last step is for object which support neither indexed nor named properties (though I'm now pretty sure you're right and that's the intent), or if it's for those that do, but fell through the cracks, e.g. if O supports indexed properties _but_ P is _not_ an array index property name (not addressed by step 1, unclear what should happen to it).